build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,361 @@
+apiVersion: v1
+data:
+  Install-WindowsCNI-Containerd.ps1: |
+    $ErrorActionPreference = "Stop";
+    mkdir -force c:/var/log/antrea
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force c:/opt/cni/bin/
+    mkdir -force c:/etc/cni/net.d/
+    cp $mountPath/k/antrea/cni/* c:/opt/cni/bin/
+    cp $mountPath/etc/antrea/antrea-cni.conflist c:/etc/cni/net.d/10-antrea.conflist
+    mkdir -force c:/k/antrea/bin
+    cp $mountPath/k/antrea/bin/antctl.exe c:/k/antrea/bin/antctl.exe
+  PostStart-AntreaAgent-Containerd.ps1: |
+    $networkName = "antrea-hnsnetwork"
+    $networks = Get-HnsNetwork
+    $networkExists = $false
+    foreach ($network in $networks) {
+        if ($network.Name -eq $networkName) {
+            $networkExists = $true
+            break
+        }
+    }
+    if ($networkExists) {
+        Enable-VMSwitchExtension -Name "Open vSwitch Extension" -VMSwitchName "antrea-hnsnetwork"
+    }
+  Run-AntreaAgent-OVS-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
```

Comment:
After pod restart, will the old $mountPath paths still remain in $env:PATH?

Reply:
I don't think so because we haven't set the PATH at Machine or User level.

Reply:
`$mountPath/openvswitch/usr/sbin` is not needed in $env:Path in agent container

Reply:
removed it, previously included because files were also present.